Makefile

```diff
@@ -193,23 +193,19 @@ storage:
 
 .PHONY: kafka
 kafka:
+	@echo Creating namespace $(KAFKA_NAMESPACE)
+	@kubectl create namespace $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
 ifeq ($(OLM),true)
 	@echo Skipping kafka-operator deployment, assuming it has been installed via OperatorHub
 else
-	@echo Creating namespace $(KAFKA_NAMESPACE)
-	@kubectl create namespace $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
 	@sed 's/namespace: .*/namespace: kafka/' ./test/kafka-operator.yml | kubectl -n $(KAFKA_NAMESPACE) apply -f -  2>&1 | grep -v "already exists" || true
-	@kubectl apply -f ./test/kafka.yml -n $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
 endif
+	@kubectl apply -f ./test/kafka.yml -n $(KAFKA_NAMESPACE) 2>&1 | grep -v "already exists" || true
 
 .PHONY: undeploy-kafka
 undeploy-kafka:
-ifeq ($(OLM),true)
```

Comment:
Sorry to be difficult, but I think I would still prefer to undeploy the operator explicitly - for example it will include CRDs that would not be cleaned up. So just add ifeq/else/endif between the two `kubectl delete` commands below.